docs/index.md

```diff
@@ -5,74 +5,50 @@ id: introduction
-npm install --save @superfaceai/one-sdk
-```
-
-Create `index.js` file, and insert the following code:
-
-```js
-const { SuperfaceClient } = require('@superfaceai/one-sdk');
-const sdk = new SuperfaceClient();
-
-async function main() {
-  // Load the profile identified by profile ID and version
-  const profile = await sdk.getProfile('vcs/user-repos@2.0.1');
-
-  // Load use case, pass use case name as argument
-  const useCase = profile.getUseCase('UserRepos');
-
-  // Invoke the use case, pass user in perform input parameter and provider name in perform options
-  const result = await useCase
-    .perform({
-      user: 'superfaceai', // use case input, in our case GitHub user name
-    }, {
-      provider: 'github' // provider name
-    });
-
-  // Handle the result
-  try {
-    const data = result.unwrap();
-    console.log(data);
-  } catch (error) {
-    console.error(error);
-  }
-}
-
-main();
-```
-
-Run it:
-
-```shell
-node index.js
-```
-
-:::tip
-
-Check out [how Superface works](./how-superface-works.mdx) to learn more about what goes on under the hood.
-
-Or read [getting started](./getting-started.mdx) for a more detailed step-by-step guide.
-
+Superface is the AI for APIs. It makes creating API integrations faster, and more management so you can focus on developing your application.
+
+## Benefits of using Superface
+
+<div className="row padding-bottom--lg">
+  <div className="col col--6">
+    <div className="card shadow">
+      <div className="card__body">
+        <h3>💸 Developing integrations over and over is expensive</h3>
+        <p>Use integrations developed by others, just as you'd use npm packages or crates.
+        </p>
+      </div>
+    </div>
+  </div>
+
+<div className="col col--6">
+    <div className="card shadow">
+      <div className="card__body">
+        <h3>🔐 Your data is safe</h3>
+        <p>Superface isn't a proxy nor a middle-man. Your application connects directly to the API you want to use.
+        </p>
+      </div>
+    </div>
+  </div>
+  <div className="col col--6">
+    <div className="card shadow">
+      <div className="card__body">
+        <h3>🎓 Decoupled workflow.</h3>
+        <p>This approach gives you a framework to decouple the lifecycle of your application and the integrations it uses.
+        </p>
+      </div>
+    </div>
+  </div>
+    <div className="col col--6">
+    <div className="card shadow">
+      <div className="card__body">
+        <h3>🧐 Abstracted Integrations.</h3>
+        <p>Superface is a language and a protocol for abstracting integrations as application use cases. Allowing you to unify your integration approach in code regardless of the API provider.
+        </p>
+      </div>
+    </div>
+  </div>
+</div>
+
+:::caution Deprecation Notice
+We recently started deprecating the previous version of Superface CLI, OneSDK and our provider use case catalog. If you are looking for the documentation for these, check out our [Classic Docs](./classic) section.
```

Comment:
When reading this with the new coming user's mindset, I was confused what this is because the web's talking about CLI and OneSDK everywhere yet here it's deprecated. Might be useful to add specific version thresholds.

```suggestion
We recently started deprecating the previous version of Superface CLI (before 4.0.0), OneSDK (before 3.0.0, for Node.js only) and our provider use case catalog. If you are looking for the documentation for these, check out our [Classic Docs](./classic) section.
```